shared/src/main/java/uk/gov/di/authentication/shared/services/SessionService.java

```diff
@@ -113,17 +113,11 @@ public void deleteSessionFromRedis(String sessionId) {
 
     public Optional<Session> readSessionFromRedis(String sessionId) {
         try {
```

Comment:
Not sure if this is more or less nice:
```java
        String serializedSession = redisConnectionService.getValue(sessionId);
        return Optional.ofNullable(serializedSession).map(s -> {
            try {
                return OBJECT_MAPPER.readValue(s, Session.class);
            } catch (Json.JsonException e) {
                throw new RuntimeException(e);
            }
        });
```

I think I'd be tempted to define a `readValueUnchecked`, then it could just be 
```java
        String serializedSession = redisConnectionService.getValue(sessionId);
        return Optional.ofNullable(serializedSession).map(s -> OBJECT_MAPPER.readValue(s, Session.class));
```